README.md

```diff
@@ -35,3 +35,23 @@ oc adm must-gather --image=registry/username/your-built-image
 
 ## Obfuscate confidential information
 There is a dedicated effort to obfuscate and omit confidential information. Head over to [openshift/must-gather-clean](https://github.com/openshift/must-gather-clean) for more information.
+
+## Loki Log Collection
+
+### Options
+- `--only-loki` - Collect only Loki logs (skip traditional must-gather)
+- `--logs-namespace=NAME` - Target specific namespace
+- `LOKI_MAX_BATCHES=N` - Batches per log type (default: 100, each batch contain 5000 logs)
+
+### Output Structure
+```
+loki-logs/
+├── application_logs_part1.txt     (250K records)
```

Comment:
Based on what I looked I think we need to download the logs to a NDJson(new line delimited Json) files, that will be easier for external tools to consume.
How hard would is be to convert them to this format.

In addition, I think we should continue downloading and creating files and generate a gz file to compress them, so that at the end we get 1Gb of data.

Reply:
@sradco 
You mean by default remove all other must gather script if loki stack exist? 
we can use --only-loki option for that and by default collect only few MB..

about the json - also in txt files you can use grep but I can try to convert this no problem.

Reply:
@avlitman  Must Gather doesnt collect only Logs.
I mean to remove only the logs collection.
@ingvagabund  would that make sense to you?

Reply:
@ ingvagabund What is the format current logs are collect in?